WordPress/tumblr-importer
# Add WordPress-Extra Coding Standards

## Changes

Before our team dives into improving the Tumblr Importer, I noticed inconsistencies in the coding standards used in the current importer code. Function and if/else statement spacing varies, and parts of the code do not follow WordPress coding standards.

This PR implements the [WordPress-Extra ruleset](https://github.com/WordPress/WordPress-Coding-Standards) and resolves all raised errors. The plugin’s functionality remains unchanged.

## Testing Instructions
* Run this branch and test connecting from scratch. Clear any saved data from the plugin, then follow the prompts to reconnect the OAuth App.
* Verify that all screens display as expected and that imports function correctly.
